signac/contrib/project.py

```diff
@@ -1467,8 +1463,9 @@ def temporary_project(self, dir=None):
         with TemporaryProject(cls=type(self), dir=dir) as tmp_project:
             yield tmp_project
 
+    # TODO: Is there a reason to support the make_dir parameter?
```

Comment:
Do things fail if `make_dir` is false? I wasn't aware of this parameter. I think it would be fine to remove and always assume it's true.

Reply:
Yeah you get a LookupError. I think we can safely remove it as well.